go/vt/binlog/tables_filter_test.go

```diff
@@ -90,9 +89,7 @@ func TestTablesFilterSkip(t *testing.T) {
 	})
 	_ = f(eventToken, statements)
 	want := `position: "MariaDB/0-41983-1" `
-	if want != got {
-		t.Errorf("want %s, got %s", want, got)
-	}
+	assert.Equal(t, want, got, "binlogConnStreamer.tablesFilterFunc(): got:\n%+v\nwant:\n%+v", got, want)
```

Comment:
We should remove the `got:\n%+v\nwant:\n%+v` output, since `testify` prints the same kind of output anyway.